cmd/aro/mirror.go

```diff
@@ -122,7 +130,12 @@ func mirror(ctx context.Context, log *logrus.Entry) error {
 
 	for _, ref := range mirrorImages {
 		log.Printf("mirroring %s -> %s", ref, pkgmirror.Dest(dstAcr+acrDomainSuffix, ref))
-		err = pkgmirror.Copy(ctx, pkgmirror.Dest(dstAcr+acrDomainSuffix, ref), ref, dstAuth, nil)
+		if srcAuthGeneva != nil {
+			err = pkgmirror.Copy(ctx, pkgmirror.Dest(dstAcr+acrDomainSuffix, ref), ref, dstAuth, srcAuthGeneva)
+		} else {
+			err = pkgmirror.Copy(ctx, pkgmirror.Dest(dstAcr+acrDomainSuffix, ref), ref, dstAuth, nil)
+		}
+
```

Comment:
```suggestion
		err = pkgmirror.Copy(ctx, pkgmirror.Dest(dstAcr+acrDomainSuffix, ref), ref, dstAuth, srcAuthGeneva)
```

You don't need to check `if srcAuthGeneva != nil`, because if it _is_ `nil`, you want to pass `nil`. Therefore you can still just pass `srcAuthGeneva`, which would be `nil`.

Reply:
I'll commit your suggestion.  Reasoning why I went this route is because `srcAuthGeneva` is a very specific var name.  Adding the if/else makes it more human readable, but the end-result is still the same.